File: src/main/java/org/example/mod/client/ExampleModClient.java
package org.example.mod.client;

import felis.kittens.core.client.ClientEntrypoint;
import org.example.mod.ExampleMod;

public class ExampleModClient implements ClientEntrypoint {
    @Override
    public void onClientInit() {
        ExampleMod.LOGGER.info("Client has been initialized");
    }
}

File: src/main/kotlin/org/example/mod/client/ExampleModClient.kt
package org.example.mod.client

import felis.kittens.core.client.ClientEntrypoint
import felis.side.OnlyIn
import felis.side.Side
import org.example.mod.ExampleMod

@OnlyIn(Side.CLIENT)
object ExampleModClient : ClientEntrypoint {
    override fun onClientInit() {
        ExampleMod.logger.info("Client has been initialized")
    }
}
